module-4/backend/week_two.md

```diff
@@ -1,22 +1,54 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What is Webpack and why is it useful?
+  Webpack is a build tool that contains a number of packages (babel, jQuery, mocha, chai, etc...). It works like the asset pipeline in a rails project. It compiles all your assets, finds any dependencies, and creates one .js file that is ready for production.
+
 2. When do you want to use event delegation?
+  You would use event delegation in order to have a "parent watch its children" for events. Instead of putting the event listener on the child, you'd put it on the parent. This will allow the bubbled up events to get to the parent, which can then delegate what to do.
```

Comment:
yup! this is extremely useful when we want an element that's dynamically loaded onto the page to be bound to a specific event 👍